cachito/workers/config.py

```diff
@@ -181,12 +184,23 @@ def validate_celery_config(conf, **kwargs):
         )
 
     default_env_vars = conf.cachito_default_environment_variables
-    for value in default_env_vars.values():
-        if not isinstance(value, dict):
+    for pkg_manager_value in default_env_vars.values():
+        if not isinstance(pkg_manager_value, dict):
             raise ConfigError(
                 'The configuration "cachito_default_environment_variables" must be a '
                 "dictionary of dictionaries"
             )
+        for env_var_info in pkg_manager_value.values():
+            if not isinstance(env_var_info, dict):
+                raise ConfigError(
+                    'The configuration "cachito_default_environment_variables" must be a '
+                    "dictionary of dictionaries of dictionaries!"
```

Comment:
Say that five times fast!

Reply:
I did and my dictionary jumped off the shelf.